CHANGELOG.md

```diff
@@ -1,4 +1,20 @@
-## 2.1.0 / 2018-1-19
+## 2.2.0-rc.0 / 2018-02-12
+
+* [CHANGE] Set consul server default to `localhost:8500`.
+* [CHANGE] Rename file SD mtime metric.
+* [CHANGE] Change fule file formatting.
```

Comment:
what is `fule file formatting.` ?

Reply:
rule file formatting :smile:

Reply:
I even did few google/github searches, thinking I am missing something important :smile_cat: